### PR TITLE
Meeting summaries for 2016-06-30

### PR DESCRIPTION
**Pull request by G1lius**

### CODE REVIEW

File: _posts/en/meetings/2016-06-30-meeting.md [hunk 1]
```diff
@@ -0,0 +1,121 @@
...
+
+## Segwit
+ 
+### background
+ 
+Developers are working on a soft fork to introduce segregated witness onto Bitcoin mainnet. Segregated witness (segwit) allows transaction signature data to be stored outside of the data hashed to produce transaction identifiers, removing all known forms of third-party malleability, allowing full nodes to compile the current UTXO set without downloading all signatures, and laying the groundwork for fraud proofs that can allow lightweight (SPV) clients to help enforce more of the consensus rules. The segwit soft fork also allows miners to substitute 1 byte of block space with 4 bytes of segwit data, increasing transaction capacity for wallets that use segwit. segregated witness BIPs: [BIP141][], [BIP142][], [BIP143][], [BIP144][] and [BIP145][]
+
+### meeting comments
+
+During his [review](https://petertodd.org/2016/segwit-consensus-critical-code-review) petertodd found a mempool DoS risk due to malleated transactions ([issue 8279](https://github.com/bitcoin/bitcoin/issues/8279)).
```
Comment (btcdrak) [inline _posts/en/meetings/2016-06-30-meeting.md]: found a *potential* DoS risk, it hasnt been validated yet.

File: _posts/en/meetings/2016-06-30-meeting.md [hunk 2]
```diff
@@ -0,0 +1,121 @@
...
+
+### meeting conclusion
+
+- Brainstorm about connection management stuff
+
+## Dbcache
+
+### background
+ 
+Gmaxwell noticed very slow reindexing while testing with default dbcache, which got confirmed by sipa who saw similar behavior. This was brought up in [last weeks meeting](https://bitcoincore.org/en/meetings/2016/06/23/#perceived-validation-slowdowns).
```
Comment (btcdrak) [inline _posts/en/meetings/2016-06-30-meeting.md]: links can be relative without the domain prefix. /en/meetings/2016/06/23/#perceived-validation-slowdowns

File: _posts/en/meetings/2016-06-30-meeting.md [hunk 3]
```diff
@@ -0,0 +1,121 @@
...
+{% include _toc.html %}
+ 
+- [Link to this week logs](https://botbot.me/freenode/bitcoin-core-dev/2016-06-30/?msg=68899079&page=2)
+- [Meeting minutes by meetbot](http://www.erisian.com.au/meetbot/bitcoin-core-dev/2016/bitcoin-core-dev.2016-06-30-19.01.html)
+ 
+---
+ 
+## Notes / short topics
+
+- A number of people are testing segwit + [BIP 152][] on testnet. There's no PR for this yet as this allows for testing interactions with non-segwit-BIP152 and it needs some BIP changes (in [BIP144][], [BIP 152][] or a separate BIP). It would be good to have it in before the activation of segwit, as otherwise compact blocks will suddenly disable.
```
Comment (MarcoFalke) [inline _posts/en/meetings/2016-06-30-meeting.md]: The link to Bip152 appears empty on the live page

Comment (btcdrak) [inline _posts/en/meetings/2016-06-30-meeting.md]: fixed. thanks.

**Pull request merged**